README.md

```diff
@@ -21,13 +21,121 @@ An Open Payments server runs two sub-systems, a resource server which exposes AP
 
 ## Local development
 
-### Dependencies
+
 
 - Python >= 3.11
+
+  To install python visit [Python Download](https://www.python.org/downloads/)
 - Poetry
+  To install poetry visiit [Poetry Documentation](https://python-poetry.org/docs/)
 
 ### Installation
 
+1. Activate your virtual emvironment. No need to create one, Poetry creates one. 
+   [Read Poetry documentation](https://python-poetry.org/docs/managing-environments/)  to read how to activate
+
+
+2. Install the dependencies in the poetry.lock
+
 ```
 > poetry install
 ```
+## Usage 
+To use this sdk, you will first need to install it in your project.  Currently you will need to build from source but once it is hosted on pypi you will be able to install it with pip
+
+```bash
+python3 -m pip install open-payments-python-sdk #currently not setup
+```
+## Installing from source
+
+Clone the repository 
+```bash
+git clone https://github.com/interledger/open-payments-python-sdk.git
+cd open-payments-python-sdk
+```
+
+Build the package
+```bash
+poetry build
+```
+After running this command, the wheel package will be written to the `dist/` folder in the repo you just cloned
+
+Install it in your project
+
+```bash
+pip install </path/to/>open-payments-python-sdk/dist/open_payments_sdk-0.1.0-py3-none-any.whl
```

Comment:
We do have a poetry based setup, there is no need to point to a pinned version of the wheel file.